data/json/recipes/armor/storage.json

```diff
@@ -256,7 +247,7 @@
     "qualities": [ { "id": "SEW", "level": 1 } ],
     "components": [
       [ [ "leather", 4 ], [ "rag", 4 ] ],
-      [ [ "string_6", 3 ], [ "thread", 10 ], [ "sinew", 10 ], [ "plant_fibre", 10 ], [ "yarn", 10 ] ]
+      [ [ "string_6", 3 ], [ "withered", 6 ], [ "straw_pile", 6 ], [ "thread", 10 ], [ "sinew", 10 ], [ "plant_fibre", 10 ], [ "yarn", 10 ] ]
```

Comment:
This and many entries below also need the change.
Not necessary here, but you could also clean them up: if they allow 10 filament, they should not require 3 strings but 1.